samples: buck: rework BUCK example into 4 separate ones (#74)
NordicSemiconductor/npmx-zephyr: samples/buck_output_simple/README.rst

.. _buck_output_simple_sample:

> BUCK - set simple output voltage
> ################################

Reviewer: ```suggestion
BUCK - Simple output voltage setting
####################################
```

.. contents::
   :local:
   :depth: 2

The BUCK Output Simple sample demonstrates how to control PMIC BUCKs using npmx drivers.

Requirements
************

The sample supports the following development kits:

.. table-from-sample-yaml::

The sample also requires an nPM1300 EK.

Overview
********

> This sample uses the npmx BUCK driver to simple set output voltage for the buck converter.

Reviewer 2: ```suggestion
This sample uses the npmx BUCK driver to set the output voltage for the buck converter.
```

> The voltage is set to 2.4 V (:c:enumerator:`NPMX_BUCK_VOLTAGE_2V4`) on BUCK1.

Reviewer: ```suggestion
You can set the output voltage to 2.4 V (:c:enumerator:`NPMX_BUCK_VOLTAGE_2V4`) on BUCK1.
```
Please verify.
